android/app/src/main/java/im/status/ethereum/MainActivity.java

```diff
@@ -202,6 +203,18 @@ private void rejectRootedNotification() {
         editor.commit();
     }
 
+    private static final String BLANK_PREVIEW = "BLANK_PREVIEW";
```

Comment:
hmm, the same constant declared twice kind of defeat the purpose of having a constant in the first place. Maybe we can just make a static class like `Constants` or `Preferences.BLANK_PREVIEW` and use it in both places.

Reply:
I think we can do without the constant. Changed the implementation a little bit in update that is coming shortly.